README.md

```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
 
 F3D is developed by a team of passionate devs. Please use F3D, star it on github or even become a [sponsor](https://github.com/sponsors/f3d-app) to support us!
 
-## Contributing
+# Goals/Non-goals
+
+We believe that clearly stating the goals and non-goals of F3D would help potential users and developers figure out if F3D is the right tool for them.
```

Comment:
Is this sentence useful?

Reply:
not that much. I just did not feel right to go directly to goals. I can remove it though.

Reply:
Let's wait for other feedback but I'd rather go straight to the point in the main readme.

Reply:
Maybe you can skip the whole "goals/non-goals" terminology as this is user facing communication. Could try something less formal like naming the section "Functionality", "Scope", "Vision", or "Philosophy" and simply saying:

> As a minimalist 3D viewer F3D aims to:
> - Support as many 3D file formats as possible
> - ...
>
> but there are no plans to:
> - Provide a classic mouse-based UI, with menus and buttons
> - ...

Reply:
I like that!